opl/cluster_read.py

```diff
@@ -309,7 +310,8 @@ def __next__(self):
                     return instance.measure(self.start, self.end, **self.config[i])
                 except Exception as e:
                     logging.error(f"Failed to measure {self.config[i]['name']}: {e}")
-                    traceback.print_exc()
+                    tb = traceback.format_exc()
+                    print(tb, file=sys.stderr)
```

Comment:
IMO you can just use https://docs.python.org/3/library/logging.html#logging.exception

Reply:
Probably you forgot to push the changes.